File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/PrehistoricMixture.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item;

/* JADX INFO: loaded from: classes3.dex */
public class PrehistoricMixture extends Item {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.item_prehistoric_mixture_name;
        this.idDescription = R.string.item_prehistoric_mixture_description;
        this.idImage = R.drawable.prehistoric_mixture;
        this.source.add(Integer.valueOf(R.string.dungeon_name_lost_lands));
        this.price = 315L;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/PrehistoricMixture.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item

class PrehistoricMixture : Item() {
    override fun configureProperties() {
        idName = R.string.item_prehistoric_mixture_name
        idDescription = R.string.item_prehistoric_mixture_description
        idImage = R.drawable.prehistoric_mixture
        source.add(R.string.dungeon_name_lost_lands)
        price = 315L
    }
}
